docker/assertj/src/main/java/com/github/dockerjava/assertions/DockerJavaAssert.java

```diff
@@ -31,6 +35,12 @@ public ContainerAssert container(String name) {
       return new ContainerAssert(container);
    }
 
+   public CubeOutputAssert withContainer(String name) {
+         String[] cmd = {"bash", "-c", "for i in $(ps -axo pid --no-headers); do echo `ps -p $i -o comm=`; done"};
```

Comment:
@lordofthejars I think `sh` is better than `bash`. It'll have only problem with windows.
 WDYT?